modal/image.py

```diff
@@ -2021,6 +2027,24 @@ def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
             dockerfile_function=build_dockerfile,
         )
 
+    def cmd(self, cmd: list[str]) -> "_Image":
+        """Set the default entrypoint argument (`CMD`) for the image.
+
```

Comment:
Should we clarify how this is used in a Modal context here?

Reply:
Good question. I started doing just that, but felt it'd be weird to have it in here before the functionality (and the image builder version on which it depends) is released.